cmd/dockerd/daemon.go

```diff
@@ -264,11 +265,22 @@ func (cli *DaemonCli) start(opts daemonOptions) (err error) {
 	// Notify that the API is active, but before daemon is set up.
 	preNotifySystem()
 
-	d, err := daemon.NewDaemon(cli.Config, registryService, containerdRemote)
+	pluginStore := plugin.NewStore(cli.Config.Root)
```

Comment:
I'm not too familiar with the `Config.Root` - but did we want this in `<root>/plugins` or is the root expected?